addons/udes_stock/models/stock_inventory.py

```diff
@@ -26,6 +26,22 @@ class StockInventory(models.Model):
         store=False,
         compute="_compute_conflicting_theoretical_qty_message",
     )
+    u_mobile_type = fields.Selection(
+        string="Mobile Type",
+        selection=[
+            ("mobile", "Mobile"),
+            ("workflow", "Workflow"),
```

Comment:
Would it be better to add the `workflow` selection in the workflows module?

Reply:
For sure it will be needed like a field when udes_workflows is not installed, as we will filter with this field stock checks have been created from old mobile as well and udes_workflows may not be even installed.
Yes it can be done to add a selection in udes_workflows, but i think its just adding complexity as we already know the reason of using the field. We agree to be added in udes_stock